docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx

```diff
@@ -0,0 +1,260 @@
+  via Terraform
+- The `tctl` CLI, which allows you to manage Teleport resources from your local
+  computer or your CI environment
+
+## Prerequisites
+
+To follow this guide, you must have:
+
+<Tabs>
+<TabItem label="tctl">
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+</TabItem>
+<TabItem label="Kubernetes Operator">
+
+A running operator by following either:
+- [the guide to enable the operator in the `teleport-cluster` Helm chart](./teleport-operator-helm.mdx).
+- [the guide to setup a standalone operator](./teleport-operator-standalone.mdx).
+
+You must also set the namespace in which you deployed the operator as this is
+the namespace where you will deploy the CustomResources:
+
+```code
+# for operators deployed with the `teleport-cluster` Helm chart
+$ export OPERATOR_NAMESPACE="teleport-cluster"
+
+# for standalone operators
+$ export OPERATOR_NAMESPACE="teleport-iac"
+```
+
+</TabItem>
+<TabItem label="Terraform">
+
+A functional Teleport Terraform provider by following [the Terraform provider guide](./terraform-provider.mdx).
+
+</TabItem>
+</Tabs>
+- a server running OpenSSH you will add to the Teleport cluster. This server
+  must be reachable from the proxy (public IP address and firewall allowing
+  traffic on port 22).
+
+<Admonition type="tip">
+  If you want to add a private SSH server (e.g. behind a NAT, in a private
+  network, protected by a firewall blocking inbound traffic, ...) you can
+  [install a Teleport agent](../../server-access/getting-started.mdx). The
+  Teleport agent opens a tunnel to the Teleport Proxy Service, allowing any user
+  to connect to it by going through the Proxy Service.
+</Admonition>
+
+## Step 1/4 - Gather the required information
+
+To register an OpenSSH server in Teleport you will need the following information:
+
+- The server hostname: <Var name="ssh-server-hostname">
+- The public server IP address with SSH port: <Var name="198.51.100.1:22">
+
+You must also choose a set of labels for the server. Those labels can be used to
+describe the server and control which users can access the server. They can be
+dynamically changed later, without having to reconfigure openSSH.
+
+See the [Access Controls for Servers](../../server-access/rbac.mdx) page for
+more details about labels and how to control access to your servers.
+
+In the rest of this guide, the labels will be:
+```yaml
+env: test
+team: engineering
+```
+
+## Step 2/4 - Write the server manifest
+
+In this step, we'll write text files describing the OpenSSH server resource we
+want to register in Teleport. Those files are called manifests and their syntax
+will vary based on the IaC tooling you'll use.
+
+Those manifests are typically versioned in a shared revision system like git.
+This allows you to keep track of all changes, follow standard code review
+procedures before changing resources in Teleport, and quickly redeploy your
+Teleport instance if needed.
+
+<Tabs>
+<TabItem label="tctl">
```

Comment:
For the tab order, can we prioritize Terraform, then Kube Operator, and then tctl?

Reply:
I think we started with tctl on the other pages so I'd keep the order like this for now and do a PR to reorder in the future. I think that tctl would also make sense as a first choice as this has the lowest entrance barrier compared o TF and the operator